translations/de/task_short_description.html

```diff
@@ -1 +1,2 @@
-Division, great for numbers, food, labor, and buried treasure. Not so much for politics.
\ No newline at end of file
+Teilen, großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. Nicht so sehr in der
```

Comment:
Mich stört: _großartig bei Zahlen, ..._ aber: [großartig] _nicht so sehr in der Politik_.
"Für" würde besser klingen, geht für beides. Für die Politik ist das Teilen nicht grossartig. Für Zahlen schon, auch für vergrabene Schätze. Obwohl mit den Zahlen habe ich gerade etwas Mühe, es ist ja für den Anwender großartig. Mal sehen, Teilen ist großartig für Zahlen, um Sie in kleinere und handlichere Teile zu verwandeln. Zahlen waren noch nie so lieb wie nach dem Teilen ^^ Für sich allein wäre "mit" hier am besten. Für was soll Teilen gut sein? Für Zahlen, Essen, Arbeit. Für die Arbeit großartig, schneller und effizienter erledigt. Und hätten Schätze Gefühle, fänden Sie es auch grossartig mehr unter die Leute zu kommen ^^ Mhh, vielleicht geht ja "mit" als Universallösung? Ich überlasse Dir die Entscheidung.